subworkflows/local/diamond_blastx.nf

```diff
@@ -0,0 +1,55 @@
+//
+// Runs diamond blastx
+//
+
+include { CHUNK_FASTA_BUSCO } from '../../modules/local/chunk_fasta_by_busco'
+include { UNCHUNK_BLASTX    } from '../../modules/local/unchunk_blastx'
+include { DIAMOND_BLASTX    } from '../../modules/nf-core/modules/diamond/blastx/main'
+
+workflow CHUNK_BLASTX {
+    take:
+
+    // CHUNK_FASTA_BUSCO input
+
+    /// File: Input fasta
+    fasta
+    /// File: Table for the first BUSCO lineage, it is an output from busco_diamond_blastp
+    busco_table
+
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    //
+    // Chunk input fasta file
+    //
+    CHUNK_FASTA_BUSCO (
+    fasta.map { fa -> [ [id: fa.baseName ], fa ] },
+    busco_table
+    )
+    ch_versions = ch_versions.mix(CHUNK_FASTA_BUSCO.out.versions.first())
```

Comment:
I  really don't remember why I added it, but it really doesn't make sense to keep it because there is only one emitted output (`CHUNK_FASTA_BUSCO.out.versions`), I will remove it.

Reply:
Done, removed first from the output versions channel.